src/bitcoin-cli.cpp

```diff
@@ -50,7 +50,7 @@ static void SetupCliArgs()
     gArgs.AddArg("-rpcwait", "Wait for RPC server to start", false, OptionsCategory::OPTIONS);
     gArgs.AddArg("-rpcwallet=<walletname>", "Send RPC for non-default wallet on RPC server (needs to exactly match corresponding -wallet option passed to bitcoind)", false, OptionsCategory::OPTIONS);
     gArgs.AddArg("-stdin", "Read extra arguments from standard input, one per line until EOF/Ctrl-D (recommended for sensitive information such as passphrases).  When combined with -stdinrpcpass, the first line from standard input is used for the RPC password.", false, OptionsCategory::OPTIONS);
```

Comment:
There is a `.  ` in this line as well. What makes it work here and not below?

Reply:
@MarcoFalke more precisely: the `FormatParagraph` call makes `-stdin` option output _without_ two spaces between words; that's why help2man parses it without any issue.